### DISCUSSION TITLE
How to view the debug under Docker?

### DISCUSSION THREAD

**Discussion by straussbastian** [FlowiseAI/Flowise#758] [Q&A]
I use Flowise under Docker. Where it is started automatically with the start of the container.

I'm working on custom tools right now, but I can't see if GPT is using these tools correctly or what data is being passed from the API. So far the results seem wrong.

How can I see the DEBUG in the terminal? Or where can I find the logging when I set the parameters DEBUG=true and LOG_LEVEL=debug.

**Comment by chungyau97** [comment]
Hi @straussbastian,

Your log path will be located [here](https://github.com/FlowiseAI/Flowise/blob/main/docker/.env.example#L6) in docker `LOG_PATH=/root/.flowise/logs` shown below:
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/086ebefe-6da4-4b9e-9196-6d0807fa9831)

I highly recommend you use [this](https://github.com/flowiseai/flowise#-developers) for debugging/development.

**Comment by bouchtaoui-dev** [comment]
First of all, what an amazing project!

It would be a nice feature if each component can show a result of an action. This way we can easily check if a request has succeeded or not, without diving in the log files. If more details are needed, one can always analyze the logs.

What do you guys think? Adding a log button, for seeing the result of the last action. We can also make it configurable to turn it on/off if necessary.

**Comment by straussbastian** [comment]
yes, I would wish for that too. Above all, it would be interesting to see what GPT is doing with the custom function API call and what information it has read in. I am currently testing this with https://autobahn.api.bund.dev/ and the results are simply wrong. GPT simply seals stories that do not match the API result. for me, the question is: is it my function? Or is it GPT? Your weather example works fine!

Edit: it was my function,but i can't See any erros in the log.

**Comment by chungyau97** [comment]
Hi @bouchtaoui-dev and @straussbastian,

I will close this discussions.
As #689 it will be an enhancement of accessing log files through Flowise's UI

The current workaround 
1. Access the file path shared above
2. Through terminal by setting [.env.example Line: 18](https://github.com/FlowiseAI/Flowise/blob/main/docker/.env.example#L18) shown below: [documentation](https://github.com/FlowiseAI/Flowise/tree/main/docker#usage)
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/a06c2128-583d-4ce6-8d19-c5df95e4b7ef)
